lumapps/api/jwt.py

```diff
@@ -0,0 +1,80 @@
+import jwt
+import requests
+import requests_cache
```

Comment:
Please use httpx instead of request (this should be the same api) https://www.python-httpx.org/

Request is available via a subdependency but we use httpx for http calls in the library.